src/test/java/com/wikia/webdriver/common/dataprovider/VisualEditorDataProvider.java

```diff
@@ -388,4 +195,89 @@ public static String getTestWiki(boolean isRTEext, boolean isVEext) {
       }
     }
   }
+
+  public enum Formatting {
+    PARAGRAPH(By.tagName("p")), HEADING(By.tagName("h2")), SUBHEADING1(By.tagName("h3")), SUBHEADING2(
+        By.tagName("h4")), SUBHEADING3(By.tagName("h5")), SUBHEADING4(By.tagName("h6")), PREFORMATTED(
+        By.tagName("pre"));
+
+    private By tag;
+
+    private Formatting(By tag) {
+      this.tag = tag;
+    }
+
+    public By getTag() {
+      return tag;
+    }
+
+    ;
+  }
+
+  public enum Style {
+    BOLD(By.tagName("b")), ITALIC(By.tagName("i")), STRIKETHROUGH(By.tagName("s")), UNDERLINE(By
+        .tagName("u")), SUBSCRIPT(By.tagName("sub")), SUPERSCRIPT(By.tagName("sup"));
+
+    private By tag;
+
+    private Style(By tag) {
+      this.tag = tag;
+    }
+
+    public By getTag() {
+      return tag;
+    }
+
+    ;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)